src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/dto/SaksbehandlerDto.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.tilleggsstonader.sak.opplysninger.oppgave.dto
+
+data class SaksbehandlerDto(
+    val fornavn: String?,
+    val etternavn: String?,
+    val rolle: SaksbehandlerRolle,
+)
+
+/*  I tilfeller hvor saksbehandler manuelt oppretter en revurdering eller en førstegangsbehandling vil oppgaven
+ *  som returneres fra oppgavesystemet være null. Dette skjer fordi oppgavesystemet bruker litt tid av variabel
+ *  lengde på å opprette den tilhørende behandle-sak-oppgaven til den opprettede behandlingen.
+ *
+ *  Dersom null blir returnert og behandlingen befinner seg i steget REVURDERING_ÅRSAK,
```

Comment:
Vi har ikke REVURDER_ÅRSAK steg, tror det er en EF greie. For oss gjelder nok dette for INNGANGSVILKÅR